stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -28,6 +28,7 @@ spec:
               {
                   "name": "copy-default-config",
                   "image": "{{.Values.Master.Image}}:{{.Values.Master.ImageTag}}",
+                  "imagePullPolicy": {{ default "IfNotPresent" .Values.imagePullPolicy | quote }},
```

Comment:
I suggest removing the default and using the same variable as the Master container and have the default of IfNotPresent explicitly defined . Take a look at https://github.com/kubernetes/charts/pull/309